backend/app/controllers/backend/albums_controller.rb

```diff
@@ -87,7 +87,7 @@ def set_album
       end
 
       def set_albums
-        @albums = Album.albums(@search, @index_items_limit * @page)
+        @albums = Album.albums(params, @index_items_limit * @page)
```

Comment:
cool @simaob ! I've put it on the `ApplicationRecord` model.
If there is a better place to place it, please tell me!!